machine-runner/src/runner/runner.ts

```diff
@@ -469,12 +569,26 @@ export interface StateOpaque<
    * }
    */
   is<
+    DeductMachineName extends MachineName,
```

Comment:
these should probably be `Deduce`, right?

Reply:
Right. I keep getting the two wrong that I had to check the dictionary again

Reply:
fixed in https://github.com/Actyx/machines/pull/29/commits/85ed0b3f30efb6990caf97ab1e6de1b826f78653